### PR TITLE
i18n(pl) Pages [astro-pages.mdx]

### PR DESCRIPTION
**Pull request by RezixDev**
#### Description (required)
Updates the Polish translation of [Pages](https://docs.astro.build/pl/basics/astro-pages/) :
docs/src/content/docs/pl/basics/astro-pages.mdx
#### Related issues & labels (optional)
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/pl/basics/astro-pages.mdx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  <head>
+    <title>Moja Strona Główna</title>
+  </head>
+  <body>
+    <h1>Witaj na mojej stronie!</h1>
+  </body>
+</html>
+```
+
+Strona musi produkować pełny dokument HTML. Jeśli nie zostanie to jawnie uwzględnione, Astro domyślnie doda niezbędną deklarację `<!DOCTYPE html>` i zawartość `<head>` do każdego komponentu `.astro` zlokalizowanego w `src/pages/`. Możesz zrezygnować z tego zachowania dla poszczególnych komponentów, oznaczając je jako [częściowe](#częściowe-strony) strony.
```
Comment (woolf1717) [inline src/content/docs/pl/basics/astro-pages.mdx:56]: (#częściowe-strony) - nie działa
trzeba dać coś typu
(#strony-częściowe)

**Review by woolf1717 — CHANGES_REQUESTED** [review]

**Comment by yanthomasdev** [comment]
Hey @RezixDev, thanks for this! I can see there's a few comments/suggestions made here, could you address them? Also, I believe there have been a few updates to this page if you could take a look and update your PR 🙌

File: src/content/docs/pl/basics/astro-pages.mdx (hunk 1 above)
Comment (RezixDev) [reply src/content/docs/pl/basics/astro-pages.mdx]: Resolved Anchor Link :)

Thanks!

File: src/content/docs/pl/basics/astro-pages.mdx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+## Niestandardowa strona błędu 500
+
+Aby uzyskać niestandardową stronę błędu 500 do wyświetlenia dla stron, które są [renderowane na żądanie](/pl/guides/server-side-rendering/#włącz-renderowanie-na-żądanie-po-stronie-serwera), utwórz plik `src/pages/500.astro`. Ta niestandardowa strona nie jest dostępna dla stron renderowanych wstępnie i nie może być wstępnie renderowana.
+
+Jeśli wystąpi błąd podczas renderowania tej strony, Twojemu odwiedzającemu zostanie pokazana domyślna strona błędu 500 Twojego hosta.
+
+<p><Since v="4.10.3" /></p>
+
+W fazie rozwoju aplikacji, przy obecności pliku `500.astro`, błąd rzucony w czasie wykonywania jest logowany w Twoim terminalu, w przeciwieństwie do pokazywania się w nakładce błędów.
```
[the remark this answers is not in the document]
Comment (RezixDev) [reply src/content/docs/pl/basics/astro-pages.mdx:113]: Same as before:

Thrown -> rzucony
Returned -> zwrócony

**Comment by RezixDev** [comment]
@yanthomasdev The Page got updated and Issues got resolved ;)

**Review by yanthomasdev — APPROVED** [review]
LGTM! Amazing work! 🙌

**Pull request merged**